src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Android.cs

```diff
@@ -213,7 +195,6 @@ public Container(Context context) : base(context)
 			{
 				set
 				{
-					_child?.DisconnectHandler();
```

Comment:
And the child is no longer getting the events.

Reply:
Same comment from here https://github.com/dotnet/maui/pull/12480/files#r1134288780

Plus, some of this code isn't really as relevant currently because we have more places where `handlers` will get disconnected. 

1) If the `MauiContext` gest changed the handler will disconnect
2) if a new handler is set on a `virtualview` the previous handler will get disconnected. 

That second one wasn't really the case when this was originally written so we don't aggressively need to disconnect here.